deploy/s3/action.yml

```diff
@@ -0,0 +1,43 @@
+name: AWS Deploy S3 Files
+description: Deploy files to an s3 bucket
+inputs:
+  source:
+    description: "source directory to upload into the s3 bucket"
+    required: true
+    default: "./dist"
+  bucket:
+    description: bucket name to deploy to
+    required: true
+  prefix:
+    description: "prefix where to upload files in bucket"
```

Comment:
I personally think that "prefix" can be misleading here since this value is a folder path inside an S3 bucket. Maybe we should use a name like `bucket-path` instead?

Reply:
I kind of know but `prefix` is the name used in the documentation and cli of `AWS`. I think keeping the same naming convention is useful for people that knows `s3`. I can improve the description to make it more clear.

https://alexwlchan.net/2020/08/s3-prefixes-are-not-directories